features/embed-video.md

```diff
@@ -0,0 +1,72 @@
+# Embed video
+
+An embedded video lets you use the video directly on your platform from Imagkit, without any additional video player. An embed video code would  usually look like this
```

Comment:
```suggestion
An embedded video lets you use the video directly on your platform from ImagKit, without any additional video player. An embed video code would usually look like this
```

Reply:
got address in [2fc2641](https://github.com/imagekitio/docs-keep-a-changelog/pull/200/commits/2fc2641830c83c76f77ab7f4c11f39527c1aeabd)